### ISSUE TITLE
Wrapping Parser Error:  MetaDataDictionary

### ISSUE THREAD

**Issue by melrobin** [InsightSoftwareConsortium/ITK#467]
[Before submitting an issue, please check that your issue hasn't been already
filed]

### Description

ITK build fails from development sources

### Steps to Reproduce

ITK build fails with the following error:
[ 56%] Generating ../../itkMetaDataObject.xml
In file included from /home/melrobin/packages/ITK/build/Wrapping/itkMetaDataObject.cxx:16:
/home/melrobin/packages/ITK/Modules/Core/Common/include/itkMetaDataObject.h:173:29: error:
      no member named 'Find' in 'itk::MetaDataDictionary'
  auto keyIter = Dictionary.Find(key);
                 ~~~~~~~~~~ ^
/home/melrobin/packages/ITK/Modules/Core/Common/include/itkMetaDataObject.h:174:30: error:
      no member named 'End' in 'itk::MetaDataDictionary'
  if ( keyIter == Dictionary.End() )
                  ~~~~~~~~~~ ^
2 errors generated.
make[2]: *** [Wrapping/Modules/ITKCommon/CMakeFiles/ITKCommonCastXML.dir/build.make:1363: Wrapping/itkMetaDataObject.xml] Error 1
make[1]: *** [CMakeFiles/Makefile2:40500: Wrapping/Modules/ITKCommon/CMakeFiles/ITKCommonCastXML.dir/all] Error 2
make: *** [Makefile:163: all] Error 2

The Find() and End() methods depend on ITK_WRAPPING_PARSER not being defined.  Somehow this value has been defined (?).

### Expected behavior
Build completion

### Actual behavior

Build error

### Reproducibility


### Versions
 Development sources


### Environment

gcc 7.3 and Fedora 27

### Additional Information

[Any additional information, configuration or data that might be necessary to reproduce the issue.]


Note: Use issues for their purpose; issues are not for code help. Need help? Ask
your question at [https://discourse.itk.org/](https://discourse.itk.org/)

**Comment by thewtex** [comment]
Thanks for the report @melrobin .

This is being addressed by #466

**Comment by thewtex** [comment]
Addressed by #478
